origin-discovery/src/listener/listener.js

```diff
@@ -304,6 +304,8 @@ const config = {
   defaultContinueBlock: parseInt(process.env.CONTINUE_BLOCK || 0)
 }
 
+logger.debug("CONFIG", JSON.stringify(config))
```

Comment:
remove since config is already logged below at line 348 ?

Reply:
Yeah this has no place in here :) deleting